src/tooling/piral-cli-webpack/src/configs/common.ts

```diff
@@ -4,11 +4,10 @@ import { progress, logReset, log } from 'piral-cli/utils';
 import { RuleSetRule, ProgressPlugin, HotModuleReplacementPlugin, optimize } from 'webpack';
 import { ImportMapsWebpackPlugin } from 'import-maps-webpack-plugin';
 import { HotModuleServerPlugin } from './HotModuleServerPlugin';
+import SheetPlugin from './sheetPlugin';
```

Comment:
Should be `./SheetPlugin` (case).

Reply:
Fixed in commit [e13cafe](https://github.com/smapiot/piral/pull/377/commits/e13cafe881af07aa1c75c07407bb9d3855fca751).